RSSReader/View/FeedViewController.m

```diff
@@ -9,31 +9,44 @@
 #import "FeedCell.h"
 #import "FeedItem.h"
 #import "UIViewController+ActivityIndicator.h"
+#import "CustomTableViewProtocol.h"
+#import "WebViewController.h"
 
-@interface FeedViewController () <UITableViewDataSource, UITableViewDelegate>
+@interface FeedViewController () <UITableViewDataSource, UITableViewDelegate, CustomTableViewProtocol>
```

Comment:
How to retry download feed if error occured?
How to dismiss alert?

https://user-images.githubusercontent.com/48133328/104765651-44fd2700-577a-11eb-9c55-6234fde6ae30.mov

Reply:
fixed